app/mailers/candidate_mailer.rb

```diff
@@ -488,6 +488,15 @@ def eoc_deadline_reminder(application_form)
     )
   end
 
+  def new_cycle_has_started(application_form)
+    @academic_year = "#{application_form.recruitment_cycle_year + 1} to #{application_form.recruitment_cycle_year + 2}"
```

Comment:
Can we use the cycle_year_range here?

```
CycleTimetable.cycle_year_range(RecruitmentCycle.next_year)
```

![image](https://user-images.githubusercontent.com/42515961/127475484-1a8a31d2-640d-4625-8349-38b2ef30559b.png)